src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -341,6 +341,7 @@ class ShenandoahFreeSet : public CHeapObj<mtGC> {
 
   void recycle_trash();
   void log_status();
+  void log_status_with_heap_locked();
```

Comment:
Suggestion: this can be just `log_status_under_lock`. We already have other methods with `*_under_lock`.

Reply:
Checked some code in this name pattern:
```
HeapWord* ShenandoahHeap::allocate_memory_under_lock(ShenandoahAllocRequest& req, bool& in_new_region) {
  // If we are dealing with mutator allocation, then we may need to block for safepoint.
  // We cannot block for safepoint for GC allocations, because there is a high chance
  // we are already running at safepoint or from stack watermark machinery, and we cannot
  // block again.
  ShenandoahHeapLocker locker(lock(), req.is_mutator_alloc());
  return _free_set->allocate(req, in_new_region);
}
```

It assumes lock is not acquired, lock is acquired in the method, I can change it to this pattern to align the naming conversions/patterns, but won't be a simple renaming.

Reply:
Ah right. Okay, nevermind, whatever works.

Reply:
I have fixed it, thanks!